src/caselawclient/Client.py

```diff
@@ -637,6 +637,9 @@ def user_has_privilege(self, username, privilege_uri, privilege_action):
 
     @cached
     def user_can_view_unpublished_judgments(self, username):
+        if self.user_has_admin_role(username):
```

Comment:
1. Does production public ui now use a non-admin user? Wary about whether this will let people potentially see unpublished documents. (Should be fine but not 100% convinced)

2. Was concerned about multiple requests to work out if a user can do a thing, but it's cached so that should be fine.

Reply:
No, production doesn't yet have the apps using their custom users, I was going to wait until after office hours to move the apps over.